### PR TITLE
Remove Auth module

### PR DESCRIPTION
**Pull request by bogh**

### CODE REVIEW

File: server/router/router.go [hunk 1]
```diff
@@ -197,22 +189,12 @@ func (router *router) HandleMessage(message *protocol.Message) error {
 }
 
 func (router *router) Subscribe(r *Route) (*Route, error) {
-	logger.WithFields(log.Fields{
-		"accessManager": router.accessManager,
-		"route":         r,
-	}).Debug("Subscribe")
+	logger.WithFields(log.Fields{"route": r}).Debug("Subscribe")
```
Comment (marian-craciunescu) [inline server/router/router.go]: use `logger.WithField("route",r)`

File: server/router/router.go [hunk 2]
```diff
@@ -225,10 +207,7 @@ func (router *router) Subscribe(r *Route) (*Route, error) {
 
 // Subscribe adds a route to the subscribers. If there is already a route with same Application Id and Path, it will be replaced.
 func (router *router) Unsubscribe(r *Route) {
-	logger.WithFields(log.Fields{
-		"accessManager": router.accessManager,
-		"route":         r,
-	}).Debug("Unsubscribe")
+	logger.WithFields(log.Fields{"route": r}).Debug("Unsubscribe")
```
Comment (marian-craciunescu) [inline server/router/router.go]: same as above

**Review by cosminrentea — APPROVED** [review]

File: server/router/router.go (hunk 2 above)
Comment (bogh) [reply server/router/router.go]: Done.

File: server/router/router.go (hunk 1 above)
Comment (bogh) [reply server/router/router.go]: Done

**Comment by bogh** [comment]
@marian-craciunescu Done both :dancer:

**Review by marian-craciunescu — APPROVED** [review]

**Pull request merged**